db/seeds.rb

```diff
@@ -1,7 +1,51 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+class Seed
+  def self.start
+    new.generate
+  end
+
+  def generate
+    create_categories
+    create_items
+  end
+
+  def create_categories
+    titles = ["Stylez", "Grub", "Weapons", "Survival Gear"]
+    titles.each do |title|
+      Category.create(title: title)
+    end
+  end
+
+  def create_items
+    Item.create!([
+      {
+      title: "axe1",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.